docs/_specification/1.2-RC1/contextual-entities.md

```diff
@@ -505,9 +505,9 @@ This example shows how to define a place, using a [geonames] ID:
 }
 ```
 
-{% include callout.html type="tip" content='To find the `@id` and `identifier` corresponding to a GeoNames HTML page like <https://www.geonames.org/8152662/catalina-park.html>, click its `.rdf` button to download the RDF metadata (<https://sws.geonames.org/8152662/about.rdf>). In the RDF metadata, find the line that looks like the following: 
-`<gn:Feature rdf:about="http://sws.geonames.org/8152662/">`. The part in the quotes is the identifier (in this case, <http://sws.geonames.org/8152662/>) .
-' %}
+{% include callout.html type="tip" content="To find the `@id` and `identifier` corresponding to a GeoNames HTML page like <https://www.geonames.org/8152662/catalina-park.html>, click its `.rdf` button to download the RDF metadata (<https://sws.geonames.org/8152662/about.rdf>). In the RDF metadata, find the line that looks like the following: 
+`<gn:Feature rdf:about=\"http://sws.geonames.org/8152662/\">`. The part in the quotes is the identifier (in this case, <http://sws.geonames.org/8152662/>)."
```

Comment:
this doesn't quite work with the `\"`, unfortunately
![full-styled spec works ok](https://github.com/user-attachments/assets/f8dc01d7-33fe-4a04-b8b0-13b13bd88363)
![one-page HTML includes the backslashes](https://github.com/user-attachments/assets/fbaf626c-3d44-4016-a971-c0d2d6ed0dfb)